PyDSS/pyControllers/Controllers/PvController.py

```diff
@@ -186,8 +186,8 @@ def CPFcontrol(self):
         """
         PFset = self.__Settings['pf']
         PFact = self.__ControlledElm.GetParameter('pf')
-        Ppv = abs(sum(self.__ControlledElm.GetVariable('Powers')[::2]))/ self.__Srated
-        Qpv = -sum(self.__ControlledElm.GetVariable('Powers')[1::2])/ self.__Srated
+        Ppv = abs(sum(self.__ControlledElm.GetVariable('Powers')[::2])) / self.__Srated
```

Comment:
This file reads bus values. That may have a big effect if there are thousands of PVSystems. Could you read all of the bus values and pass them in?

Reply:
I'm not actually sure if this comment is correct. Does the active PVSystem get reset if you set a bus value?

Reply:
Does the active PVSystem get reset if you set a bus value. No it does not